src/js/api/config.js

```diff
@@ -104,7 +105,7 @@ const Config = function(options, persisted) {
     let rateControls = config.playbackRateControls;
 
     if (rateControls) {
-        let rates = [0.5, 1, 1.25, 1.5, 2];
+        let rates = config.playbackRates;
```

Comment:
This takes care of setting the default rates, but would fail to sanitize the rates if `playbackRates` contains array items that aren't rounded or aren't within the min/max values allowed. Eg.:

```
config.playbackRates = [0.5, 1.25, 3.23, 5];
```

You'll want to check if `rateControls` is an array and assign it to `rates`, then filter `rates`.